pkg/sql/plan/build_show.go

```diff
@@ -447,6 +447,12 @@ func buildShowStatus(stmt *tree.ShowStatus, ctx CompilerContext) (*Plan, error)
 	return returnByRewriteSQL(ctx, sql, ddlType)
 }
 
+func buildShowCollation(stmt *tree.ShowCollation, ctx CompilerContext) (*Plan, error) {
+	ddlType := plan.DataDefinition_SHOW_COLLATION
+	sql := "select 'utf8mb4_bin' as `Collation`, 'utf8mb4' as `Charset`, 46 as `Id`, 'Yes' as `Compiled`, 1 as `Sortlen`"
+	return returnByRewriteSQL(ctx, sql, ddlType)
```

Comment:
i don't known if you want to support like/where ?

Reply:
> 

for now only 1 collation is supported, so only 1 row was returned. like/where is only supported in parser now, and dont actually work. will add it later when more collations supported